Gemfile.jruby-2.3.lock.release

```diff
@@ -49,7 +49,7 @@ GEM
     aws-sdk-v1 (1.67.0)
       json (~> 1.4)
       nokogiri (~> 1)
-    backports (3.8.0)
+    backports (3.10.3)
```

Comment:
3.8.0 was from April of this year and is represents a multi-month jump.  I remember this exact question for different review. 

@robbavey - Do you to remember discussions around backports version jump ?  (or was it someone else?)

Reply:
@jakelandis I do recall it - the same thing happened when we bumped to 5.6.4 (https://github.com/elastic/logstash/pull/8470/files). I remember we had a conversation about it - which I thought was here, but doesn't look like it. 

The backports gem version history (https://rubygems.org/gems/backports/versions) shows it to have been completely unchanged from April up until October, when there was a flurry of versions committed over a period of 10 days.